portfolio/src/main/java/com/google/sps/servlets/LoginServlet.java

```diff
@@ -61,23 +63,25 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     status = true;
     person.addProperty("url", logoutUrl);
 
-    /* 
-    make this section print onto the DOM from the js file
+    // make this section print onto the DOM from the js file
+
+    // out.println("<h1>Home</h1>");
+    // out.println("<p>Hello " + nickname + "!</p>");
+    // out.println("<p>Logout <a href=\"" + logoutUrl + "\">here</a>.</p>");
+    // out.println("<p>Back to home page <a href=\"/index.html\">here</a>.</p>");
 
-    out.println("<h1>Home</h1>");
-    out.println("<p>Hello " + nickname + "!</p>");
-    out.println("<p>Logout <a href=\"" + logoutUrl + "\">here</a>.</p>");
-    out.println("<p>Change your nickname <a href=\"/nickname\">here</a>.</p>");
-    */
 
     person.addProperty("status", status);
     person.addProperty("name", nickname);
-    // person.addProperty("name_url", "/nickname");
+    person.addProperty("home", "/index.html");
```

Comment:
Do you expect this to vary from user to user?  I would expect the home page to always default to index.html